### PR TITLE
Added 3D source for Kingbright KPBD-3224 package.

### PR DESCRIPTION
**Pull request by cnieves1**
Datasheet: http://www.kingbright.com/attachments/file/psearch/000/00/20160808bak/KPBD-3224SURKZGC(Ver.8A).pdf

### CODE REVIEW

**Comment by myfreescalewebpage** [comment]
@cnieves1 where does this file comes from ? Have you created it yourself ?

**Comment by cnieves1** [comment]
I know. This 3D was created using Unigraphics (now Siemens) NX software.
Original source in this PR was the result of exporting to step and importing into FreeCAD.
As it was not the source, I deleted the FreeCAD file and uploaded the original source (NX file).
I know this is a propietary SW, but it is the original source...
Is this allowed or not?

**Comment by myfreescalewebpage** [comment]
It is not because of licence issues. You should push a FreeCAD source file made by yourself.

**Comment by chschlue** [comment]
@cnieves1 The important question is: Did **you** create that file?

**Comment by cnieves1** [comment]
@chschlue yes, does it change anything?

**Comment by chschlue** [comment]
Of course, copyright is about who, not how (in layman's terms, IANAL).
I noticed you kept using passive voice ("was created using"), both here and over at the model PR, that's why I asked directly.

Proprietary tools are permissible, although open source is preferred. This also means that your model (and source) may be replaced in the future (if somebody contributes an open alternative).
